near/contracts/token-bridge/src/lib.rs

```diff
@@ -974,6 +974,15 @@ impl TokenBridge {
         if namount == 0 {
             env::panic_str("EmptyTransfer");
         }
+
+        if nfee >= namount {
```

Comment:
Everywhere else you make this check last.  Why is it first here?  (Sorry OCD kicking in...)
Does it even matter?

Reply:
no particular reason...